### ISSUE TITLE
[Dashboard] New /teams view

### ISSUE THREAD

**Issue by jldec** [gitpod-io/gitpod#15101]
With the removal of the old team plans, the website and docs require a new landing page (at /teams) which shows your Gitpod teams (the same list as the dashboard menu dropdown).

The page should have an action to create a new team. When the list of existing teams is empty, this "New team" action should be more prominent and very easy to see.

For users who are still owners of an older team-plan in their personal settings, this new /teams page should offer a link to the old team-plans page. We can rename the older page /team-plans. Team-plans will still be reachable for those users by navigating through their personal settings menu.

**Comment by jldec** [comment]
cc: @MichaelAring

**Comment by svenefftinge** [comment]
I don't think we should encourage maintaining many teams. Instead, I'd love to think through improving the UX so that teams get out of your way. As a user, I usually work in a context (i.e. team) and am interested in my workspaces and projects. I believe if we were building the UX well enough most users would ever only have one team.

Is the reason to build this solely to link to something from the website?

**Comment by jldec** [comment]
Closing this pending further evolution of teams.
